src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -387,6 +390,23 @@ private static VersionRange GetVersionRange(IVsReferenceItem item)
             return VersionRange.All;
         }
 
+        private static bool GetPropertyBoolOrFalse(
+                IVsReferenceItem item, string propertyName)
+        {
+            try
+            {
+                return MSBuildStringUtility.IsTrue(item.Properties?.Item(propertyName)?.Value);
+            }
```

Comment:
Shouldn't be possible, we are only dealing with this as a string.
https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Commands/Utility/MsBuildStringUtility.cs#L66